Dockerfile

```diff
@@ -13,7 +16,12 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
     # install required python packages
     pip install --no-cache-dir -r requirements/dev-requirements.txt && \
     # install mlflow in editable form
-    pip install --no-cache-dir -e . && \
+    pip install --no-cache-dir -e .
+
+
+# Build MLflow UI
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
```

Comment:
Why do we need this change?

Reply:
While this PR was draft, a separate [commit was made](https://github.com/mlflow/mlflow/commit/fdeb6e27a2e3449da3115768c94411d16a622d55) to tidy up the Dockerfile. I can refactor to fit the latest `master` if you're prefer?

Reply:
@funkypenguin Let's refactor!